src/applications/vaos/redux/api/vaosApi.js

```diff
@@ -0,0 +1,27 @@
+import { createApi } from '@reduxjs/toolkit/query/react';
+import { apiRequestWithUrl } from 'applications/vaos/services/utils';
+
+export const vaosApi = createApi({
+  reducerPath: 'appointmentApi',
+  baseQuery: () => ({ data: null }),
```

Comment:
What's the reason for this dummy base query that retrieves null data? I wonder if we should set the base query to something like `/vaos/v2/` since all of our APIs except [this one](https://github.com/department-of-veterans-affairs/vets-website/blob/19bf477347f482d80c9a1b6314460c71bb261183/src/applications/vaos/services/vaos/index.js#L166) uses paths with that prefix.

Reply:
@JunTaoLuo This is just because we are using a util function to actually make the API call. To use the normal fetch you can add a baseUrl here. For our app we want to use the `apiQuery` util function. Here we are using a wrapper function for `apiQuery`.

Reply:
Ah okay, it just looks a little odd so maybe we should add a comment explaining it. It certainly surprised me but maybe that's because I've never used `createApi` before.

Reply:
Okay, I'll add a comment